id/__main__.py

```diff
@@ -83,6 +83,8 @@ def main() -> None:
     else:
         print(token)
 
+def entrypoint() -> None:
+    main()
```

Comment:
This isn't strictly required as one could imaging just using

```toml
"id.__main__:main"
```

in `pyproject.toml` but this allows for subtle decoupting between `entrypoint` and `main` if desired, [like `build` has](https://github.com/pypa/build/blob/3c25681123fa1b3af58855eb43204f0de006561a/src/build/__main__.py#L444-L449).

Reply:
@di might have a different opinion, but IMO it's okay to have this just be `main` instead -- I don't think we have a plan/foreseen need for decoupling the two, so YAGNI applies 🙂

Reply:
I'm ok with it just being `main` as well!